### DISCUSSION TITLE
How to enable Inlay Hints in Neovim without rust-tools?

### DISCUSSION THREAD

**Discussion by ChenZhongPu** [rust-lang/rust-analyzer#15031] [Q&A] [answered]
Is it possible to show `Inlay Hints` in Neovim without `rust-tools`? If so, how to enable it?

**Comment by lnicola** [comment]
Looks like `rust-tools` has [a bunch](https://github.com/simrat39/rust-tools.nvim/blob/master/lua/rust-tools/inlay_hints.lua) of code to deal with those, so I'd guess you can't, without copy-pasting that.

What are you trying to achieve? Why not use `rust-tools`?

**Reply by ChenZhongPu** [reply, marked as answer]
> Why not use `rust-tools`?

Because `rust-tools` provides too many features that I may not use, and I would like a minimum & clean config/setting.

After some research, I found that [lsp-inlayhints.nvim](https://github.com/lvimuser/lsp-inlayhints.nvim) is able to do so.
